### ISSUE TITLE
Single onward journey - Headline

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#687]
**Is your feature request related to a problem? Please describe.**
The SOJ needs a headline, this should be a `<a>` element (not a heading)

**Describe the solution you'd like**
The element should have the following CSS:
- Pica typography specs
- colour: #222
- padding bottom 16px (this is different on the designs so UX need to confirm, however it is consistent as the gap between titles and timestamps across other components)

**Comment by pjlee11** [comment]
Following the decision [here](https://github.com/bbc/simorgh/issues/685#issuecomment-423241528) will inform the decision for whether this element is  ` <h3>` or a `<span>`
